spec/support/shared_examples_for_recovery_codes.rb

```diff
@@ -87,7 +87,7 @@
       context 'when the session times out and the modal is visible' do
         before do
           allow(Figaro.env).to receive(:session_check_frequency).and_return('1')
-          allow(Figaro.env).to receive(:session_check_delay).and_return('2')
+          allow(Figaro.env).to receive(:session_check_delay).and_return('1')
```

Comment:
yeah that shouldn't have made it in. It doesn't stop the spec from randomly failing either.